src/event-handler/index.ts

```diff
@@ -31,6 +31,13 @@ interface EventHandler<TTransformed = unknown> {
 export function createEventHandler(options: Options<any>): EventHandler {
   const state: State = {
     hooks: {},
+    log: {
```

Comment:
I wonder if maybe we should have a `createLogger` helper function that'll let us consolidate these, and ideally remove some of the `istanbul ignore`s?

I'm thinking something like:

```
const createLogger = (debugNamespace: string, logger: Logger | boolean): Logger => ({
  debug: logger ? () => {} : debug(`webhooks:${debugNamespace}`),
  info: console.info.bind(console),
  warn: console.warn.bind(console),
  error: console.error.bind(console),
  ...logger,
});
```

It also looks like we're not using `info` anywhere, so we could probably just omit that for now - if we wanted to use it in the future we could add it in as a non-breaking feature.

Reply:
(`createLogger` could also be helpful for us, as we could then mock it in tests to return a set of mock functions)

Reply:
Done, I've extracted `createLogger()`.

My preference is to keep `info`, for consistency with the other `@octokit/*` packages. I can think of a couple ways to pacify Istanbul, what I like about the comment is that it's obvious why it's there, and it's trivial to remove when it's no longer needed.